README.md

```diff
@@ -37,10 +37,10 @@ For now, to install these packages:
 ``` bash
 go get github.com/c-fs/vendor
 go get -u github.com/c-fs/Jerasure
+export GO15VENDOREXPERIMENT=1
```

Comment:
`GO15VENDOREXPERIMENT`  requried `go version > 1.5`.  It is better to explicit go version.